src/util/log.d

```diff
@@ -165,6 +173,25 @@ struct Log
                 }
             }
         }
+
+        static if (supportsStringInterpolation)
+        {
+            void append(Fence _ = Fence(), string file = __FILE__, size_t line = __LINE__, A...)
+                (InterpolationHeader _1, lazy A args, InterpolationFooter _2)
```

Comment:
Why not `header` and `footer` instead of `_1` and `_2`?

Reply:
done